requirements.txt

```diff
@@ -1,9 +1,10 @@
-pytorch_lightning  # placeholder
+torchmetrics
+lightning-bolts==0.3.2  # todo: we shall align with proper release
+pytorch_lightning
```

Comment:
no minimum version?

Reply:
well this is just placeholder for docs as they are mocking libs and canot handle install from URL as have just one line bellow